base/linalg/triangular.jl

```diff
@@ -1038,6 +1038,11 @@ for (f, g) in ((:/, :A_rdiv_B!), (:A_rdiv_Bc, :A_rdiv_Bc!), (:A_rdiv_Bt, :A_rdiv
         end
     end
 end
+### Fallbacks brought in from linalg/bidiag.jl while fixing #14506.
+# Eventually the above promotion methods should be generalized as
+# was done for bidiagonal matrices in #14506.
+At_ldiv_B(A::AbstractTriangular, B::AbstractVecOrMat) = At_ldiv_B!(A, copy(B))
+Ac_ldiv_B(A::AbstractTriangular, B::AbstractVecOrMat) = Ac_ldiv_B!(A, copy(B))
```

Comment:
I don't get it. That section is explaining the methods for `Bidiagonal`, but I don't see an explanation of these methods. I think that the `AbstractTriangular` part of the old definitions might have been redundant because of the definitions I link to above.